src/Middleware/HttpLogging/src/W3CLoggerOptions.cs

```diff
@@ -106,6 +106,16 @@ public TimeSpan FlushInterval
         }
     }
 
+    /// <summary>
+    /// List of additional request header values to log.
```

Comment:
Update this to talk about request headers, not just cookies (e.g. `Request Headers can contain private information which may have... Arbitrary Request Headers should not be logged unless...`

Reply:
Whatever general warnings we add about the security concerns of logging request headers here we should also add to `HttpLoggingOptions`.

Reply:
added to both places